### PR TITLE
Add man pages workflow

### PR DESCRIPTION
**Pull request by cicdguy**
Add workflow to check if man pages are up-to-date.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+.github/workflows/* @dinakar29
```
Comment (bms63) [inline .github/CODEOWNERS:1]: What is this?

**Review by bms63 — APPROVED** [review]

**Comment by bms63** [comment]
I was hoping to use template repo to set up another repo.  Is this is good enough state to do that?

File: .github/CODEOWNERS (hunk 1 above)
Comment (cicdguy) [reply .github/CODEOWNERS:1]: This means that I will be auto-assigned for a review if someone changes the `.github/workflows/*` files. More about CODEOWNERS files here: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

**Comment by cicdguy** [comment]
> I was hoping to use template repo to set up another repo. Is this is good enough state to do that?

Yes it is. Go for it.

File: .github/CODEOWNERS (hunk 1 above)
Comment (bms63) [reply .github/CODEOWNERS:1]: wow that is amazing...does it block the merging as well?

Comment (bms63) [reply .github/CODEOWNERS:1]: ill make you an admin as well.

Comment (cicdguy) [reply .github/CODEOWNERS:1]: It can if you set up branch protection rules that way.

Comment (bms63) [reply .github/CODEOWNERS:1]: okay.  i made you admin and set up branch protection for main.

**Comment by rossfarrugia** [comment]
thanks @dinakar29 and @bms63 for all your efforts on this repo! i'll set up admiralophtha now too using this

**Pull request merged**